tensorflow/python/kernel_tests/array_ops/array_ops_test.py

```diff
@@ -1927,7 +1927,7 @@ class QuantizeAndDequantizeV2Test(test_util.TensorFlowTestCase):
 
   def testInvalidArgs(self):
     with self.assertRaisesRegex((errors.InvalidArgumentError, ValueError),
-                                r"(.*should be scalar.*|.*Broadcast dimension -1 is out of bound.*)"):
+                                r"(.*should be scalar.*|.*out of bound.*)"):
       q, _, _ = array_ops.quantize_and_dequantize_v2(
           input=np.ones((10)),
           input_min=[-1, 1],
```

Comment:
I meant these -1's, so that the two paths are failing for the "should be scalar" reason.  i.e. replace them with a 2 or something so the inputs are within bounds, but that there should only be a single value.